### PR TITLE
(DAL) WS consumer script for easy testing and logging

### PR DESCRIPTION
**Pull request by nick-bisonai**
# Description

subscribe and read ws results 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

File: node/script/test_dal_consumer_ws/main.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+		Method: "SUBSCRIBE",
+		Params: []string{},
+	}
+
+	for _, configs := range configs {
+		subscription.Params = append(subscription.Params, "submission@"+configs.Name)
+	}
+
+	wsEndpoint := fmt.Sprintf("ws://dal.%s.orakl.network/ws", chain)
+	// wsEndpoint := "ws://localhost:8090/ws"
```
Comment (Intizar-T) [inline node/script/test_dal_consumer_ws/main.go:40]: should we remove this if not used?

File: node/script/test_dal_consumer_ws/main.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	jsonMarshalData, err := json.Marshal(data)
+	if err != nil {
+		return err
+	}
+	err = json.Unmarshal(jsonMarshalData, &wsData)
+	if err != nil {
+		return err
+	}
+
+	// log.Info().Any("wsData", wsData).Msg("ws message")
```
Comment (Intizar-T) [inline node/script/test_dal_consumer_ws/main.go:84]: same here 🙏

**Review by Intizar-T — APPROVED** [review]
lgtm!

File: node/script/test_dal_consumer_ws/main.go (hunk 1 above)
Comment (nick-bisonai) [reply node/script/test_dal_consumer_ws/main.go:40]: I left the comment so that I can use it easily, for personal use, but if you insist, will remove 😅

**Pull request merged**